scripts/ensure_ollama_resources.py

```
@@ -0,0 +1,127 @@
+    try:
+        resp = httpx.get("http://localhost:11434/api/tags", timeout=10)
+        resp.raise_for_status()
+        models = resp.json().get("models", [])
+        return [m.get("name", "") for m in models if m.get("name")]
+    except Exception:
+        return []
+
+
+def pull_model(model: str) -> Tuple[bool, str]:
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)